openedx/adg/admin_dashboard/performance/helpers.py

```diff
@@ -0,0 +1,58 @@
+"""
+Helper methods for `performance` app
+"""
+from lms.djangoapps.grades.models import PersistentCourseGrade
+from openedx.adg.admin_dashboard.performance.constants import COURSE_GRADES_REPORT_HEADER
+from openedx.adg.common.helpers import convert_float_point_to_percentage
+
+
+def get_course_grades_report_data(users, grades):
+    """
+    Returns data for the csv report for course grades.
+
+    Args:
+        users (list): List of users for which report data will be generated.
+        grades (dict): Dict containing course grades information.
+
+    Returns:
+        list: List containing data for the csv report.
+    """
+    rows = [COURSE_GRADES_REPORT_HEADER]
+    for user in users:
+        rows.append(
+            [
+                user.id,
+                user.profile.name,
+                user.email.encode('utf-8'),
+                user.profile.phone_number,
+                user.username.encode('utf-8'),
+                grades[user.id]['grade_percentage'],
+                user.course_enrollment[0].created.date(),
```

Comment:
`course_enrollment[0]` can this list ever be empty? Also can we use a descriptive variable for this?

Reply:
Nope, This will never be empty as all the user here will have an active course enrolment for the course and that enrolment is prefetched.